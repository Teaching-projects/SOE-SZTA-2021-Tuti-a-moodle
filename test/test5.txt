90
50
20
100
60
30