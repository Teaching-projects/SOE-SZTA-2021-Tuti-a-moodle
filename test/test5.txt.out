Enter the name of the entity: Enter the lore of A: Enter the health of A: Enter the attack of A: Enter the defense of A: Enter the cooldown of A: 
Enter the name of the entity: Enter the lore of B: Enter the health of B: Enter the attack of B: Enter the defense of B: Enter the cooldown of B: 
====================
A - HP: 90.0, DMG: 50.0, DEF: 20.0
B - HP: 90.0, DMG: 50.0, DEF: 20.0
====================
The Battle begins.
A and B clash!
====================
The Battle is over.
Both heroes are dead!